XpringKit/PayID/XRPPayIDClientProtocol.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
```

Comment:
Just FYI in the future you can do:
```
$ git mv old_file.swift new_file.swift
```

And file version history / diffs will be kept around.

Reply:
gold - wrote it down in my git notes